app/presenters/audit_presenter.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class AuditPresenter
```

Comment:
feels weird that this is AuditPresenter and then the helpers are nested under Audit and not under AuditPresenter